src/generators/SpirvShaderGenerator.cpp

```diff
@@ -109,11 +115,24 @@ void CSpirvShaderGenerator::Generate()
 	auto mainFunctionLabelId = AllocateId();
 
 	WriteOp(spv::OpCapability, spv::CapabilityShader);
+	WriteOp(spv::OpCapability, spv::CapabilityInt8);
```

Comment:
These should be added only if we are using uint8 inside the shader. We can do something like `hasInvocationInterlock` for that.